blockchain/indexers/common.go

```diff
@@ -27,6 +32,16 @@ var (
 	// errInterruptRequested indicates that an operation was cancelled due
 	// to a user-requested interrupt.
 	errInterruptRequested = errors.New("interrupt requested")
+
+	// indexTipsBucketName is the name of the db bucket used to house the
+	// current tip of each index.
+	indexTipsBucketName = []byte("idxtips")
+
+	// noPrereqs indicates no index prerequisites.
+	noPrereqs = "none"
+
+	// bufferSize represents the index notification buffer size.
+	bufferSize = 1
```

Comment:
It looks like you may have been doing some testing with a buffer size of 1 and forgot to change it back.

Reply:
Ahh missed it again, resolving.